frontends/web/src/routes/lightning/components/payment-details.tsx

```diff
@@ -0,0 +1,141 @@
+/**
+ * Copyright 2024 Shift Crypto AG
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { useEffect, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import type { Payment as IPayment, LnPaymentDetails } from '@/api/lightning';
+import { Dialog } from '@/components/dialog/dialog';
+import { FiatConversion } from '@/components/rates/rates';
+import { TxDetail } from '@/components/transactions/components/detail';
+import { TxDateDetail } from '@/components/transactions/components/date';
+import { TxDetailCopyableValues } from '@/components/transactions/components/address-or-txid';
+import { getTxSign } from '@/components/transactions/utils';
+import styles from '@/components/transactions/components/details.module.css';
+
+type TTxDetailsDialog = {
+  open: boolean;
+  onClose: () => void;
+  payment: IPayment;
+  sign: string;
+}
+
+export const PaymentDetailsDialog = ({
+  open,
+  onClose,
+  payment,
+  sign,
+}: TTxDetailsDialog) => {
+  const { t } = useTranslation();
+
+  return (
+    <Dialog
+      open={open}
+      title={t('transaction.details.title')}
+      onClose={onClose}
+      slim
+      medium>
+      {payment && (
+        <>
+          <TxDetail label="Memo">
+            {payment.description}
+          </TxDetail>
+          <TxDateDetail
+            time={new Date(payment.paymentTime * 1000).toString()}
+          />
+          <TxDetail label="Amount">
+            {payment.amountMsat * 0.001} sat
```

Comment:
There is `toSat()` fn available for this in `utils/conversion.ts`, which adds rounding